docs/content/integrations/dbt.mdx

```diff
@@ -10,69 +10,52 @@ description: Dagster can orchestrate dbt alongside other technologies.
+
+@asset(compute_kind="tensorflow", non_argument_deps={"daily_order_summary"})
+def predicted_orders():
+    ...
+```
 
 ---
 
-## dbt and Dagster software-defined assets tutorial
-
-In this tutorial, we'll walk you through integrating dbt with Dagster using dbt's example [jaffle shop project](https://github.com/dbt-labs/jaffle_shop), the [dagster-dbt library](/\_apidocs/libraries/dagster-dbt), and a data warehouse, such as [DuckDB](https://duckdb.org/).
-
-By the end of the tutorial, you'll have a working dbt and Dagster integration and a handful of materialized Dagster assets, including a plotly chart powered by data computed from your dbt models.
-
-<ArticleList>
-  <ArticleListItem
-    title="Overview and prerequisites"
-    href="/integrations/dbt/using-dbt-with-dagster"
-  ></ArticleListItem>
-  <ArticleListItem
-    title="Part one: Set up a dbt project"
-    href="/integrations/dbt/using-dbt-with-dagster/part-one"
-  ></ArticleListItem>
-  <ArticleListItem
-    title="Part two: Load dbt models as Dagster assets"
-    href="/integrations/dbt/using-dbt-with-dagster/part-two"
-  ></ArticleListItem>
-  <ArticleListItem
-    title="Part three: Create and materialize upstream Dagster assets"
-    href="/integrations/dbt/using-dbt-with-dagster/part-three"
-  ></ArticleListItem>
-  <ArticleListItem
-    title="Part four: Create and materialize a downstream asset"
-    href="/integrations/dbt/using-dbt-with-dagster/part-four"
-  ></ArticleListItem>
-</ArticleList>
+## Getting started
+
+There are a few ways to get started with Dagster and dbt:
+
+- Take the [tutorial](/integrations/dbt/using-dbt-with-dagster). We'll walk you through setting up dbt and Dagster together on your computer, using dbt's example [jaffle shop project](https://github.com/dbt-labs/jaffle_shop), the [dagster-dbt library](/\_apidocs/libraries/dagster-dbt), and a data warehouse, such as [DuckDB](https://duckdb.org/). By the end, you'll have a working dbt and Dagster project and a handful of materialized Dagster assets, including a chart powered by data from your dbt models.
+- Play around with a [working dbt + Dagster project](https://github.com/dagster-io/dagster/tree/master/examples/assets_dbt_python).
+- Browse the [dagster-dbt integration reference](/integrations/dbt/reference) for short lessons on Dagster + dbt topics.
+- Review the [API docs](/\_apidocs/libraries/dagster-dbt) for the dagster-dbt library.
 
 ---
 
-## References
-
-<ArticleList>
-  <ArticleListItem
-    title="dagster-dbt integration reference"
-    href="/integrations/dbt/reference"
-  ></ArticleListItem>
-  <ArticleListItem
-    title="dagster-dbt API reference"
-    href="/\_apidocs/libraries/dagster-dbt"
-  ></ArticleListItem>
-</ArticleList>
+## Understanding the relationship between dbt models and Dagster Software-defined Assets
```

Comment:
This title is on the long side - can we shorten it up? Suggestions:

- Relating dbt models to Dagster Software-defined Assets
- Understanding how dbt models relate to Dagster Software-defined assets
- dbt models and Dagster Software-defined Assets (not my favorite, but it's the shortest)

Reply:
I like "Understanding how dbt models relate to Dagster Software-defined assets"